### DISCUSSION TITLE
Dynex Neuropool.net and mineradnow.space

### DISCUSSION THREAD

**Discussion by slappy6212** [RainbowMiner/RainbowMiner#2436] [Q&A]
Hi,
Someone have those 2 pools .ps1 files for Dynex Neuropool.net and mineradnow.space ?

**Comment by RainbowMiner** [comment]
I will have a look into these pools and add the modules.

**Reply by RainbowMiner** [reply]
Neuropool is now part of our pool's library: commit 90250a2ef32bb13aba34d3f8c6bc9189091239d5

**Comment by slappy2** [comment]
Also can you add DNX to herominers and remove ETF (no longer supported) at the same times in herominers.
As a temp solution can you tell me how can i update it manually. I tried to edit the ps1 and the pools.config.txt myself  but no luck.
Thanks

**Reply by RainbowMiner** [reply]
Done commit bcf00c163d0384ae5d56fe7cf9f30658afde17b7

**Reply by slappy2** [reply]
You have a small typo error to rectify at the ETHW line 31. its showing ethf instead of ethw. in herominers.ps1
thanks

**Reply by slappy2** [reply]
I updated the 4 files you provided but it still only benchmarking Ekapool for Dynex...no Herominer for Dynex.

**Reply by RainbowMiner** [reply]
If Ekapool is more profitable, then it will not use Herominers. Please look at "All Pools" on http://localhost:4000 - then select algorithm "DynexSolve" and check if Herominers is available. If not, have you added `"DNX": "$DNX",` in pools.config.txt, Herominers section?

**Reply by RainbowMiner** [reply]
> You have a small typo error to rectify at the ETHW line 31. its showing ethf instead of ethw. in herominers.ps1
thanks

Oh! It is fixed, now.

**Reply by RainbowMiner** [reply]
Look:
![image](https://github.com/RainbowMiner/RainbowMiner/assets/39437538/62ee8deb-cf1e-4bc1-9665-67ab6cf9d094)

.. all three pools show up in "All Pools"

**Reply by slappy2** [reply]
oups i added the wrong info in poolconfig...it was "DNX-Params": "", insted of "DNX": "$DNX"....trial and error LOL Its there now. Thanks

**Reply by slappy2** [reply]
Hey i can see that you have Neuropool.ps1 ! is it available for download ?

**Reply by RainbowMiner** [reply]
Yes, in fact.
Look here: https://github.com/RainbowMiner/RainbowMiner/commit/90250a2ef32bb13aba34d3f8c6bc9189091239d5

Here are the files:
[./Balances/Neuropool.ps1](https://github.com/RainbowMiner/RainbowMiner/blob/master/Balances/Neuropool.ps1)
[./Data/PoolsConfigDefault.ps1](https://github.com/RainbowMiner/RainbowMiner/blob/master/Data/PoolsConfigDefault.ps1)
[./Data/poolsinfo.json](https://github.com/RainbowMiner/RainbowMiner/blob/master/Data/poolsinfo.json)
[./Data/regions.txt](https://github.com/RainbowMiner/RainbowMiner/blob/master/Data/regions.json)
[./Data/regions2.txt](https://github.com/RainbowMiner/RainbowMiner/blob/master/Data/regions2.json)
[./Pools/Neuropool.ps1](https://github.com/RainbowMiner/RainbowMiner/blob/master/Pools/Neuropool.ps1)

On each of the file's pages you will find a button "download raw". 
![image](https://github.com/RainbowMiner/RainbowMiner/assets/39437538/603ef886-e01b-47de-8026-c5f576125e86)
Click it and download in the corresponding folders.
You need to update the regions.json/regions2.json too, because I had to add MiddleEast as stratum region for that pool.
